src/app/student-login/student-login.component.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../variables";
+
+
+.page{
+    background-repeat: no-repeat;
+    background-image: url('/assets/images/large-display/hill-compressed.jpg');
+    height: 100%;
+    width: 100%;
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.login-form{
+  min-width: 40%;
+  max-width: 60%;
+  max-height: 50%;
```

Comment:
```suggestion
  width: 400px;
  margin: 16px;
```

The `width` here is essentially a maximum width for the card, it still scales down on narrower screens but also doesn't get super wide on larger screens. The margin gives a margin around it for smaller screens. See this comparison on an iPhone SE:

| Current | Suggested |
|--|--|
| ![buzz-about-git-login-page mn-pollinators vercel app_join-session(iPhone 5_SE) (1)](https://user-images.githubusercontent.com/1300395/86498163-36a16d80-bd4a-11ea-8b7d-8280b560db30.png) | ![localhost_4200_join-session(iPhone 5_SE)](https://user-images.githubusercontent.com/1300395/86498165-428d2f80-bd4a-11ea-8d65-d53875e8ee24.png) |

Reply:
Ooh, I like the look of that!